decidim-admin/app/commands/decidim/admin/create_participatory_process_admin.rb

```diff
@@ -35,19 +35,39 @@ def call
 
       attr_reader :form, :participatory_process
 
-      def create_participatory_process_admin
-        ParticipatoryProcessUserRole.create!(
-          role: :admin,
-          user: user,
-          participatory_process: @participatory_process
-        )
+      def create_role
+        if form.roles.include?("collaborator")
+          user.roles << "collaborator" unless user.roles.include?("collaborator")
```

Comment:
Well, I don't know. It doesn't make sense to show a way to invite participatory process collaborators via the UI if they are not scoped to that participatory process. This could cause privacy issues and worsen an (already existing) bug.

We should fix it on this PR.